src/main/java/org/maxgamer/quickshop/shop/VirtualDisplayItem.java

```diff
@@ -93,6 +93,10 @@ public VirtualDisplayItem(@NotNull Shop shop) throws RuntimeException {
 
     private void initFakeDropItemPacket() {
 
+        if (shop.isLeftShop()) {
+            shop.getAttachedShop().refresh();
+            return;
+        }
```

Comment:
I recommend put this code to the statement which invoking `initFakeDropItemPacket` method, since this method is doing initFakeDropItemPacket

Reply:
After extensive testing, turns out this whole statement is unnecessary!

Reply:
So why it not being removed?

Reply:
Because I am dumb and accidentally undid it before committing lmao.